test/cmd/process.sh

```diff
@@ -6,7 +6,7 @@ trap os::test::junit::reconcile_output EXIT
 (
   set +e
   oc delete all,templates --all
-  oc delete user someval
+  oc delete user someval someval=moreval someval=moreval2 someval=moreval3
```

Comment:
i'm a little surprised "=" is even a valid character in a username, but ok cool.